src/connectors/rating-menu/connectRatingMenu.js

```diff
@@ -232,7 +232,9 @@ export default function connectRatingMenu(renderFn, unmountFn = noop) {
       _toggleRefinement(helper, facetValue) {
         const isRefined =
           this._getRefinedStar(helper.state) === Number(facetValue);
-        helper.clearRefinements(attribute);
+        helper.setState(
+          helper.state.removeDisjunctiveFacetRefinement(attribute)
+        );
```

Comment:
Not that clean, but removeDisjunctive doesn’t exist on the helper directly iirc (but to double check)